Documentation/ContentObjects/Pageview/Index.rst

```diff
@@ -0,0 +1,329 @@
+
+This is a basic definition of a :fluid:`PAGEVIEW` object with only one path to the
+templates:
+
+..  code-block:: typoscript
+    :caption: EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript
+
+    page = PAGE
+    page.10 = PAGEVIEW
+    page.10.paths.100 = EXT:my_sitepackage/Resources/Private/Templates/
+
+The content of the folder could look like this:
+
+..  directory-tree::
+    :level: 2
+
+    *   EXT:my_sitepackage/Resources/Private/Templates/
+
+        *   Layouts
+
+            *   Default.html
+            *   WithoutHeader.html
+
+        *   Pages
+
+            *   Default.html
+            *   StartPage.html
+            *   TwoColumns.html
+            *   With_sidebar.html
+
+        *   Partials
+
+            *   Footer.html
+            *   Sidebar.html
+            *   Menu.html
+
+If a page has a backend layout named "with_sidebar", the template file is
+then resolved
+to :file:`EXT:my_sitepackage/Resources/Private/Templates/Pages/With_sidebar.html`.
+
+If the backend layout is named "TwoColumns" it is resovled to
+:file:`EXT:my_sitepackage/Resources/Private/Templates/Pages/TwoColumns.html`.
+
+For all these templates
+:ref:`partials <t3viewhelper:render_partial>`
+are expected in folder
+:path:`EXT:my_sitepackage/Resources/Private/Templates/Pages/Partials` and
+:ref:`layouts <t3viewhelper:typo3fluid-fluid-layout>` in
+:path:`EXT:my_sitepackage/Resources/Private/Templates/Pages/Layouts`.
+
+..  _cobj-pageview-paths-example-extended:
+
+Example: Define fallbacks for a template paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use the directories defined in :confval:`pageview-paths` to
+define fallback directories for the templates:
+
+.. code-block:: typoscript
+
+    page = PAGE
+    page {
+        10 = PAGEVIEW
+        10.paths {
+            100 = EXT:my_basic_sitepackage/Resources/Private/Templates/
+            200 = EXT:my_general_sitepackage/Resources/Private/Templates/
+            300 = EXT:my_special_sitepackage/Resources/Private/Templates/
+        }
+    }
+
+The template for a page with a certain backend layout is first searched in
+:path:`EXT:my_special_sitepackage/Resources/Private/Templates/Pages/` then in
+:path:`EXT:my_general_sitepackage/Resources/Private/Templates/Pages/` and last
+in :path:`EXT:my_special_sitepackage/Resources/Private/Templates/Pages/`.
+
+..  _cobj-pageview-variables:
+
+variables
+---------
+
+..  confval:: variables.[variable_name]
+    :name: pageview-variables
+    :type: :ref:`Content Object <cobject>`
+
+    Sets variables that should be available in Fluid.
+
+Example: Make additional variables available in the Fluid template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+..  literalinclude:: _includes/_pageWithVariables.typoscript
+    :langugage: typoscript
+    :caption: EXT:my_sitepackage/Configuration/TypoScript/setup.typoscript
+
+The following variables are now available in the Fluid template:
```

Comment:
Here the same... it isn't it a pageview template instead of a fluid template? Or is it a fluidtemplate just set up by a page template?

Reply:
"Fluid" is the markup language. "PAGEVIEW" is the name of the TypoScript object